model/cluster_info.go

```diff
@@ -12,7 +12,7 @@ type ClusterInfo struct {
 	Id         string `json:"id"`
 	Version    string `json:"version"`
 	ConfigHash string `json:"config_hash"`
-	IpAddress  string `json:"ipaddress"`
+	IPAddress  string `json:"ipaddress"`
```

Comment:
Please ensure that there is no issue running this on a cluster with older nodes, so that we can give SRE team prior heads up if needed.

I don't think there will be, since the `json` attribute is the same. But good to check.

Reply:
I can confirm that the JSON payload send over the wire is the same with and without this change. I didn't test a cluster with older nodes as I didn't find an easy way to set this up. Does my testing cover your concerns?

Reply:
Yes, I can see that the JSON payload will not change. This was more of a sanity check.

You can just spin up docker containers and run separate versions. But it's okay to leave it if that is taking time. Thanks Ben.